cas/cas.go

```diff
@@ -74,8 +74,8 @@ func NewStore(base string) *Store {
 	return ds
 }
 
-// tmpFile creates a temporary file in $basepath/tmp
-func (ds Store) tmpFile() (*os.File, error) {
+// TmpFile creates a temporary file in $basepath/tmp
```

Comment:
@kelseyhightower But it's only being used within `package cas` (and arguably it is an internal detail) so it doesn't need to be exported